src/clientversion.cpp

```diff
@@ -103,7 +103,7 @@ const std::string CLIENT_DATE(BUILD_DATE);
 std::string FormatVersion(int nVersion)
 {
     if (nVersion % 100 < 25)
-        return strprintf("%d.%d.%d-beta%d", nVersion / 1000000, (nVersion / 10000) % 100, (nVersion / 100) % 100, (nVersion % 100)+1);
```

Comment:
Additionally also the case of build number being 0 should be handled (since this is the build number for `main` branch).
Something like:
```
...
if (nVersion % 100 == 0)
  return strprintf("%d.%d.%d-alpha", nVersion / 1000000, (nVersion / 10000) % 100, (nVersion / 100) % 100);
if (nVersion % 100 > 0 && nVersion % 100 < 25)
...
```

Reply:
Done